src/test/java/com/shurjopay/ShurjopayTest.java

```diff
@@ -165,13 +159,15 @@ private void fillupAndSubmitPaymentForm(String url, boolean shouldFail) throws I
 	}
 	
 	/**
-	 * Prepares chrome driver instance.
+	 * Prepares chrome driver instance
+	 *
 	 * @return prepared chrome driver.
 	 */
 	private WebDriver getChrome() {
-		System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/chromedriver");
+		System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/chromedriver.exe");

```

Comment:
exe is hardcoded here. what will now happen for Linux or macos ????

Reply:
Now chrome driver loaded dynamically based on user OS.